hardware/Comply-to-the-Regulations.md

```diff
@@ -0,0 +1,394 @@
+ - The manufacturer of the end product (OEM) is responsible for the product being compliant.Silicon Labs strongly recommends to the OEM to perform radiated testing on the end product. The conducted test results can be inherited from Silicon Labs modules’ test reports. Typically, test evidence (reports) must be presented Optionally, NB (Notified Body) opinion can be asked as a formal “certificate”; 
+ - A product is tested according to harmonized standards: A harmonized standard is a European standard developed by a recognized European Standards Organization: CEN, CENELEC, or ETSI. It is created following a request from the European Commission to one of these organizations. Manufacturers, other economic operators, or conformity assessment bodies can use harmonized standards to demonstrate that products, services, or processes comply with relevant EU legislation.
+ There are RF, EMC, and Safety relevant standards for radio products. All harmonized standards with status [information](https://portal.etsi.org/webapp/WorkProgram/Frame_WorkItemList.asp?qSORT=DIRECTIVES&qHARMONIZED=True&butPredefined=Search&qDIRECTIVE=2014%2F53%2FEU&optDisplay=ALL&butHarmonized=Search);
+ - If harmonized standards are not followed, it is mandatory to use Notified Body;
+ - Manufacturer can use Notified Body also just to get more confidence about the compliance;
+ - Manufacturer maintains Technical Construction File (TCF)
+   - Test reports
+   - Description of the device (schematic, layout, BOM, photos, etc.)
+   - DoC
+   - NB opinion letter (if applied)
+ 
+# 4. Basic RF test items and limits for complying the regulations 
+Allthough there are RF, EMC and Safety categories need to be certified for regional regulations. It only cares about the RF category in this chapter. And Silicon Labs has SoCs,SiP,and PCB module products, the certification procedure or items are different among them. In this chapter, it only describes the test items and limits base on the end product with SoCs. For end products with SiP/PCB-Module, the finnal test items maybe reduced due to pre-certificated SiP/PCB-Module as described in previous section. As Silicon Labs' products contain sub-G frequency bands and 2.4GHz bands, it will introduce the test items and limits seperately in frequency bands due to the difference regulation requirements.
+## 4.1. FCC requirements in Sub-GHz bands
+Silicon Labs' products can be mainly applied to FCC-Part-15.231 and FCC-Part-15.247 regulations for sub-G bands. It regulates 260 to 470MHz frequency band in Part-15.231, and 902 to 928MHz frequency band in Part-15.247.
+- Part-15.231
+  Typical applications are Door openers, Alarm systems, Remote Switches, and Control links in this regulation. There are severy typical frequency bands as 315MHz, 433MHz, and 470MHz.
+  - Tx power limits
```

Comment:
The viewed markdown does not have a return line, which should be inserted

Reply:
Corrected.